### PR TITLE
Add a spinner on the picture while the message status is Pending

### PR DESCRIPTION
**Pull request by Flaburgan**

### CODE REVIEW

File: axolotl-web/src/assets/images/loading.svg [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!--  SVG by Sam Herbert (@sherb), for everyone. More @ http://goo.gl/7AJzbL  -->
```
Comment (Flaburgan) [inline axolotl-web/src/assets/images/loading.svg:1]: This is the same SVG that in `index.html` but I needed it as an image file this time.

File: axolotl-web/src/components/Message.vue [hunk 2]
```diff
@@ -241,6 +241,18 @@ video,
...
+.outgoing .attachment-img {
+  background: center center no-repeat;
+  background-color: #000;
+  background-image: url("../assets/images/loading.svg");
+
+  img {
+    opacity: 0.2;
+  }
+}
+.sent .attachment-img img {
```
Comment (Flaburgan) [inline axolotl-web/src/components/Message.vue:253]: Please confirm that `message.IsSent` is kept even after read and delivered. That's what I observed but I want a confirmation.

**Comment by Flaburgan** [comment]
Todo: remove the background when loaded for transparent image

File: axolotl-web/src/components/Message.vue (hunk 2 above)
Comment (nanu-c) [reply axolotl-web/src/components/Message.vue:253]: Is Sent isn't changed after sending a message so you can rely on it.

File: axolotl-web/src/assets/images/loading.svg (hunk 1 above)
Comment (nanu-c) [reply axolotl-web/src/assets/images/loading.svg:1]: that's fine

**Review by nanu-c — APPROVED** [review]

**Review by nanu-c — APPROVED** [review]

**Comment by nanu-c** [comment]
looks good to me

**Pull request merged**